apprentice_learner/views.py

```diff
@@ -135,6 +155,16 @@ def create(http_request):
                                        "ensure provided args are "
                                        "correct.")
 
+    global active_agent, active_agent_id, dont_save
+    if(active_agent != None):
+        active_agent = None
+        active_agent_id = None
+        dont_save = False
+    if(str(data.get("stay_active",False)).lower() == 'true'):
```

Comment:
So the stay active system is only turned on at create time if you want it? The don't save bit being always on in the same way seems potentially problematic if we ever run into an issue of training multiple agents at once. Most of our simulations don't do that but my RumbleBlocks implementation can do it.